### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by zach-planet-nine**

### CODE REVIEW

File: src/rust/crate/README.md
Comment (FssAy) [inline src/rust/crate/README.md:1]: In the Rust crate README there are two paragraphs about "what is Sessionless". Is it correct or should we remove one?

File: docs/CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# How to contribute
+
+I'm of the opinion that contributing should be as easy and inviting as possible. 
+This guide is written with that premise in mind.
+
+The discussion about Sessionless takes place on Discord in the [Open Source Force][osf] server. 
+Follow that link to get an invite.
```
Comment (FssAy) [inline docs/CONTRIBUTING:7]: There is a missing invitation link "https://opensourceforce.net/discord"

File: docs/CONTRIBUTING
Comment (FssAy) [inline docs/CONTRIBUTING:1]: The file is missing the `.md` extension so it's not rendered by default. I am not sure if that's intentional as there is no markdown formatting other than the `#` headers.

**Review by FssAy — CHANGES_REQUESTED** [review]
The `Methods` tables in every implementation are the same. Should we update them to comply with each implementation's source code, or is it more of a "pseudo code" to showcase the functionality of Sessionless? Nonetheless the tables should start and end with `|` symbol due to compatibility reasons _(at least that's what my IDE told me lol)_.

File: docs/CONTRIBUTING (hunk 1 above)
Comment (zach-planet-nine) [reply docs/CONTRIBUTING]: The link is the Open Source Force above, which will actually be a link now that this is a md file

File: docs/CONTRIBUTING
Comment (zach-planet-nine) [reply docs/CONTRIBUTING:1]: Thanks. The github docs are confusing.

**Comment by zach-planet-nine** [comment]
> The `Methods` tables in every implementation are the same. Should we update them to comply with each implementation's source code, or is it more of a "pseudo code" to showcase the functionality of Sessionless? Nonetheless the tables should start and end with `|` symbol due to compatibility reasons _(at least that's what my IDE told me lol)_.

Added the pipe. And yeah the methods tables sort of a pseudocode placeholder. I could give you a justification for that, but it's really I didn't feel like going into all of the implementations and updating them here, but now that you've brought it up it seems reasonable to have a note about that.

**Review by FssAy — APPROVED** [review]
👌

**Pull request merged**